enhancements/api-review/gcp_user_defined_labels.md

```diff
@@ -544,8 +546,12 @@ func() {
   operator. In this proposal, the changes proposed and developed will be part of
   openshift-* namespace. External operators are not in scope.
   User-defined tags can be updated on the following GCP resources.
-  - Compute Engine Instances
```

Comment:
The installer configuration for tags and labels provides default base values to be used on resource. The enhancement should not mandate on whether operator reconciliation. It is best to incorporate override methods in operator based on usecases similar to machine api.

Reply:
We will have to reconsider the reconciliation part during GA, which may be required for AWS, Azure labels support too.

Reply:
Ok, sounds good.

/lgtm
/approve